src/ssl.c

```diff
@@ -985,8 +985,10 @@ int wolfSSL_SetTmpDH(WOLFSSL* ssl, const unsigned char* p, int pSz,
     if (pSz < ssl->options.minDhKeySz)
         return DH_KEY_SIZE_E;
 
+    #ifndef WOLFSSL_WPAS
```

Comment:
Changed wpa_supplicant to set accept state if there are DH parameters to set.
This means I can remove the define check.